### PR TITLE
Added environment variables required for the lightstep opentracing

### PR DESCRIPTION
**Pull request by nuwandiw**
Fixes https://github.com/zalando-stups/taupage/issues/566

### CODE REVIEW

File: build/setup.d/55-setup-taupage-yaml.sh [hunk 1]
```diff
@@ -35,3 +35,33 @@ fi
...
+if [ -n "$OPENTRACING_LIGHTSTEP_COLLECTOR_HOST" ]; then
+    echo "opentracing_lightstep_collector_host: $OPENTRACING_LIGHTSTEP_COLLECTOR_HOST" >> /meta/taupage.yaml
+fi
+if [ -n "$OPENTRACING_LIGHTSTEP_COLLECTOR_PORT" ]; then
+    echo "opentracing_lightstep_collector_port: $OPENTRACING_LIGHTSTEP_COLLECTOR_PORT" >> /meta/taupage.yaml
+fi
+if [ -n "$OPENTRACING_MAX_BUFFERED_SPANS" ]; then
+    echo "opentracing_max_buffered_spans: $OPENTRACING_MAX_BUFFERED_SPANS" >> /meta/taupage.yaml
+fi
+if [ -n "$ACCOUNT" ]; then
```
Comment (aermakov-zalando) [inline build/setup.d/55-setup-taupage-yaml.sh]: These values are dynamic, they shouldn't be hardcoded during Taupage build time.

File: runtime/etc/init/planb-tokeninfo.conf [hunk 2]
```diff
@@ -14,8 +14,20 @@ script
         export OPENID_PROVIDER_CONFIGURATION_URL="$config_openid_provider_configuration_url"
         export REVOCATION_PROVIDER_URL="$config_revocation_provider_url"
         export BUSINESS_PARTNERS="$config_tokeninfo_business_partners"
+        export OPENTRACING_LIGHTSTEP_ACCESS_TOKEN="$config_opentracing_lightstep_access_token"
+        export OPENTRACING_LIGHTSTEP_COMPONENT_NAME="$config_opentracing_lightstep_component_name"
+        export ARTIFACT="$config_artifact"
+        export OPENTRACING_LIGHTSTEP_COLLECTOR_HOST="$config_opentracing_lightstep_collector_host"
+        export OPENTRACING_LIGHTSTEP_COLLECTOR_PORT="$config_opentracing_lightstep_collector_port"
+        export OPENTRACING_MAX_BUFFERED_SPANS="$config_opentracing_max_buffered_spans"
+        export ACCOUNT="$config_account"
```
Comment (aermakov-zalando) [inline runtime/etc/init/planb-tokeninfo.conf]: Should be fetched from the EC2 metadata service.

File: runtime/etc/init/planb-tokeninfo.conf [hunk 3]
```diff
@@ -14,8 +14,20 @@ script
...
+        export ZONE="$config_zone"
```
Comment (aermakov-zalando) [inline runtime/etc/init/planb-tokeninfo.conf]: Should be fetched from the EC2 metadata service.

File: runtime/etc/init/planb-tokeninfo.conf [hunk 4]
```diff
@@ -14,8 +14,20 @@ script
...
+        export DEPLOYMENT_ID="$config_deployment_id"
```
Comment (aermakov-zalando) [inline runtime/etc/init/planb-tokeninfo.conf]: Not sure who's supposed to populate it. Senza?

Comment (nuwandiw) [reply runtime/etc/init/planb-tokeninfo.conf]: Ideally this should be the senza stack version. At the moment we expect the users to set this as an environment variable from their cloud formation yaml. 
Is there a way that I can access the stack version from here?

File: build/setup.d/55-setup-taupage-yaml.sh (hunk 1 above)
Comment (nuwandiw) [reply build/setup.d/55-setup-taupage-yaml.sh]: Removed this and used ec2 metadata from the conf file

**Comment by aermakov-zalando** [comment]
👍

**Comment by jrake-revelant** [comment]
👍

**Pull request merged**